### ISSUE TITLE
Multi-packages repository

### ISSUE THREAD

**Issue by juanpicado** [verdaccio/verdaccio#897]

**Comment by ramonornela** [comment]
https://github.com/lerna/lerna ❤️

**Comment by sergiohgz** [comment]
@ramonornela yeah, Lerna 3 will be so great (and in next rc with the separation of version and publish commands better)

**Comment by juanpicado** [comment]
In order to provide more context, @sergiohgz is working on the following:

* [Add @verdaccio/eslint-config](https://github.com/verdaccio/core/issues/3)
* [Add @verdaccio/babel-preset](https://github.com/verdaccio/core/issues/2)
* [Improve @verdaccio/eslint-config](https://github.com/verdaccio/core/issues/8)

This will be integrated into several internal projects to simplify configuration and share code. At the same time it will became a multi repository project were other core projects will be part of it soon or later (https://github.com/verdaccio/core/issues/5). 

As last step, `verdaccio` will use all this core dependencies in Verdaccio 4 (#924 ).
